### PR TITLE
Spelling mistakes

### PR DESCRIPTION
**Pull request by d-fens**

### CODE REVIEW

File: source/help/joining/index.md [hunk 1]
```diff
@@ -27,22 +27,21 @@ Almost anything you want - providing the username is available and no more than
...
-## Pay By Paypal
+## Pay By PayPal
 In the payment notes, please make sure you provide us with:
   - Your __Redbrick username__.
   - If you are a new user, we __require__ your DCU student / staff ID number.
 
 {% paypal %}
 
-## Pay by bank transfer / lodgement
+## Pay by bank transfer / lodgment
```
Comment (VoyTechnology) [inline source/help/joining/index.md]: Looks like `lodgment` is the US spelling: https://en.oxforddictionaries.com/definition/lodgement

I would keep it as `lodgement`

**Comment by VoyTechnology** [comment]
Please fork in the future :)

**Comment by butlerx** [comment]
Lodgement is correct

**Comment by VoyTechnology** [comment]
@butlerx Did you close it accidentally?

**Comment by butlerx** [comment]
Yep sorry.

File: source/help/tutorials/docker/index.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ You should get `docker-compose version: 1.10.0` or higher
   docker-compose up -d
 ```
 
-Thats is do to port 443 on your domain and it'll be your nginx  
+That's is do to port 443 on your domain and it'll be your nginx
```
Comment (butlerx) [inline source/help/tutorials/docker/index.md]: Remove the is

**Comment by d-fens** [comment]
Added two new commits based on comments. I'm not an expert at git so if issues please advise.

File: source/_posts/2016-week-9-announce.md [hunk 3]
```diff
@@ -40,14 +40,13 @@ networking could mean for you.
 - Where: TBD
 - More information here: https://www.facebook.com/events/902199496580631/
 
-John Looney from Google will be doing a talk on Burnout, convering a range of topics from
-defining it, all the way to its prevention and cure.
+John Looney from Google will be doing a talk on Burnout, covering a range of topics from defining it, all the way to its prevention and cure.
```
Comment (VoyTechnology) [inline source/_posts/2016-week-9-announce.md]: Why is this all on one line now?

File: source/_posts/week-11-news-events-1.md [hunk 4]
```diff
@@ -20,7 +20,7 @@ It's nearly Christmas time, and with the season to be jolly stepping into gear a
 <!-- more -->
 ## “What's the Bright Idea” Undergraduate Awards
 
-“What’s the Bright Idea” is an event organised by the non-profit awards organisation, The Undergraduate Awards. This event is going to be a celebration of bright, young individuals and their bright ideas.The event is on this Monday, the 5th of December in the Odessa.
+“What’s the Bright Idea” is an event organised by the non-profit awards organisation, The Undergraduate Awards. This event is going to be a celebration of bright, young individuals and their bright ideas. The event is on this Monday, the 5th of December in the Odessa.
```
Comment (VoyTechnology) [inline source/_posts/week-11-news-events-1.md]: It looks like you might have rebased this wrong. https://github.com/redbrick/static-site/commit/22dd30373a387a0843d2ca39cf2dc8ee1db353c7#diff-de9d78e2dbcaa992cdeb70f00623885aR23 has changed this to use normal quotations

File: source/_posts/2016-week-9-announce.md (hunk 3 above)
Comment (butlerx) [inline source/_posts/2016-week-9-announce.md]: Part of this pr is merging lines

**Pull request merged**